csm/conf/etc/csm/csm.conf

```diff
@@ -76,7 +76,7 @@ STATS:
         ssl_check: "false"
         interval: "10" # Flush interval in sec (Rate at which stats is stored)
         offset: "20" # offset in sec
-
+    public_auth: "true"
```

Comment:
Change the key from public_auth to auth (enable / disable)

Reply:
Updated key.